mmv1/third_party/terraform/website/docs/r/container_cluster.html.markdown

```diff
@@ -1432,6 +1436,12 @@ linux_node_config {
 
 * `enable_alts` - (Required) Whether the alts handshaker should be enabled or not for direct-path. Requires Workload Identity ([workloadPool]((#nested_workload_identity_config)) must be non-empty).
 
+<a name="nested_enterprise_config"></a>The `enterprise_config` block supports:
+
+* `desired_tier` - (Optional) Sets the tier of the cluster. Available options include `STANDARD` and `ENTERPRISE`.
+
+* `cluster_tier` - (Output) Sets the tier of the cluster. Available options include `STANDARD` and `ENTERPRISE`.
```

Comment:
The standard way to document an output-only field is to add it to the `Attributes Reference` section below that contains all the computed fields.

Also this description seems to exactly match the desired tier, is that intended?